src/test/java/com/serenitydojo/WhenCreatingObjects.java

```diff
@@ -12,4 +12,17 @@ public void creating_a_dog() {
         Assert.assertEquals(fido.getFavoriteToy(), "Bone");
         Assert.assertEquals(fido.getAge(), 5);
     }
+
+    @Test
+    public void dog_makes_noise() {
+        Dog fido = new Dog("Fido", "Bone", 5);
+
+        fido.makeNoise();
+    }
+
+    @Test
+    public void dog_is_fed() {
+        Dog fido = new Dog("Fido", "Bone", 5);
+        fido.feed();
```

Comment:
you could should use an assert here as well

```suggestion
        fido.feed();
         Assert.assertEquals(fido.isFed(), true);
```